store/sqlstore/post_store.go

```diff
@@ -1283,6 +1296,25 @@ func (s *SqlPostStore) search(teamId string, userId string, params *model.Search
 	return list, nil
 }
 
+func removeMysqlStopWordsFromTerms(terms string) (string, error) {
+	stopWords := make([]string, len(searchlayer.MYSQL_STOP_WORDS))
+	copy(stopWords, searchlayer.MYSQL_STOP_WORDS)
+	re, err := regexp.Compile(fmt.Sprintf(`^(%s)$`, strings.Join(stopWords, "|")))
+	if err != nil {
+		return "", err
+	}
+
+	newTerms := make([]string, 0)
+	separatedTerms := strings.Split(terms, " ")
```

Comment:
nit: here we can use `Fields` instead and we will not need the `TrimSpace` in each term after:

```suggestion
	separatedTerms := strings.Fields(terms)
```

Reply:
Perfect!